src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -72,11 +72,37 @@ object SharedFileSystem {
   }
 }
 
+class SFSIOInterface extends IOInterface {
```

Comment:
This could probably use a better name... I think `SharedFileSystemIOInterface` is better than the abbreviated version